ocs_sample_library_preview/DataView/Field.py

```diff
@@ -168,4 +197,13 @@ def fromJson(content: dict[str, str]):
         if 'IncludeUom' in content:
             result.IncludeUom = content['IncludeUom']
 
+        if 'SummaryType' in content:
+            if content['SummaryType'] == 'None':
```

Comment:
This seems like a reasonable way to handle the ```None``` SummaryType, but our other files that use SdsSummaryType do not account for the possibility of ```None```  (See DataMapping.py and FieldMapping.py). Perhaps those files need to be updated as well

Reply:
I changed the other two classes because I don't see where they're ever tested, and I don't understand how they could work as is.